### ISSUE TITLE
Support the definition of mappings and responses in values files

### ISSUE THREAD

**Issue by tom-intermedia** [wiremock/helm-charts#44]
### Proposal

I am using deploy charts that pull together multiple different service charts and deploy them as one into many different environments.

Each environment is configured through a set of files contained in environment specific folders. I am using the folder path charts/deploy-chart/environments/environment-name/ but really these could be anywhere. 

As each environment requires different mocks to be defined I am currently using a customised version of the helm chart that allows me to specify mappings and responses inline in a values files within the environment.

It would be great if the chart could support the definition of mappings and responses via values out of the box. An additional enhancement would be to allow the location of the mapping and response folders to be specified rather than be fixed to the root of the chart..

Configmap template supporting inline mapping specification

```
apiVersion: v1
kind: ConfigMap
metadata:
  name: {{ include "wiremock.fullname" . }}-mappings-configs
  namespace: {{ .Release.Namespace | quote }}
  labels:
    app.kubernetes.io/name: {{ include "wiremock.name" . }}
    helm.sh/chart: {{ include "wiremock.chart" . }}
    app.kubernetes.io/instance: {{ .Release.Name }}
    app.kubernetes.io/managed-by: {{ .Release.Service }}
data:
  {{- if .Values.mappings }}
  {{- range $key, $value := .Values.mappings }}
  {{ $key }}: {{ $value | quote }}
  {{- end }}
  {{- else }}
  {{- $files := .Files }}
  {{- range $key, $value := .Files }}
  {{- if hasPrefix "mappings/" $key }} {{/* only when in mappings/ */}}
  {{ $key | trimPrefix "mappings/" }}: {{ $files.Get $key | quote }} {{/* adapt $key as desired */}}
  {{- end }}
  {{- end }}
  {{- end }}
```

Values file specifying mappings
```
... 
mappings: 
    some-mapping.json: |
      {
        "priority": 10,
        "request": {
          "method": "GET",
          "urlPattern": "/some-url/.*"
        },
        "response": {
          "status": 200,
          "headers": {
            "Content-Type": "application/json"
          }
        }
      }
```

**Comment by gitkent** [comment]
@tom-intermedia thanks for the idea. I think it makes sense. It makes mappings and responses more scalable. If you can raise a PR that would be good.

**Comment by tom-intermedia** [comment]
@subigre Thank you for picking this up and making the changes.

**Comment by DanielFran** [comment]
Hi @subigre @tom-intermedia 
Just to clarify, are you injecting the mappings/responses file/value via CLI or adding it directly to the value file?

**Comment by subigre** [comment]
@DanielFran Until now I used bash scripts or the test setup feature of Robot Framework to import my mappings and response files depending on the projects’ requirements. I will probably continue to use this approach for many projects in the future as it is very flexible and convenient when you have a large number of mappings. 
On the other hand, I found that @tom-intermedia's idea is also useful for use cases where I have a couple of "static" mappings or to quickly spin up a new WireMock instance for rapid testing.
